src/main/java/com/hyunec/cosmicbaseball/entity/BaseballGame.java

```diff
@@ -61,11 +77,36 @@ public String handleResult() {
             return resultHandler.handleResult("a");
         }
     }
-    private void processStrike() {
+    public void incrementStrikeCount() {
         strikeCount++;
     }
 
+    public void incrementStrikeCount(int count) {
+        strikeCount += count;
+    }
+
+    public void incrementBallCount() {
+        ballCount ++;
```

Comment:
```suggestion
        ballCount++;
```

이후 lint 관련은 언급하지 않습니다.

Reply:
구글 리팩토링 다시 적용했습니담..ㅜㅠㅠㅠ